nucypher/blockchain/eth/interfaces.py

```diff
@@ -391,7 +393,7 @@ def call_backend_verify(self, pubkey: PublicKey, signature: Signature, msg_hash:
     def unlock_account(self, address, password, duration):
         if 'tester' in self.provider_uri:
             return True  # Test accounts are unlocked by default.
-        return self.w3.personal.unlockAccount(address, password, duration)
+        return self.w3.geth.personal.unlockAccount(address, password, duration)
```

Comment:
This deserves a moment of pause.

Reply:
I see your point, and although I agree in principle, in our codebase and docs we seem to talk only about `geth`. Also, `web3.parity` doesn't seem to support locking accounts; although we're not using that functionality right now, we have a related pending TODO in `BlockchainPower`. A possible way forward is to describe this (i.e., potential support for Parity or other Ethereum clientes) as an issue.

Reply:
Following up: #933